configs/errors.eslintrc.json

```diff
@@ -0,0 +1,123 @@
+{
+  "rules": {
+    "@typescript-eslint/class-name-casing": "error",
+    "@typescript-eslint/consistent-type-definitions": "error",
+    "@typescript-eslint/indent": "off", // temp
```

Comment:
temp?

Reply:
I always had issues with my VS Code formatter, where I don't think we all share the same one. In this case, my auto-formatter wants to add indentation incompatible with this eslint rule.

Should I set the formatter options into the project's settings.json, or should I disable this rule definitely?